src/modules/stores/innovation/sections/section-4-1.config.ts

```diff
@@ -199,7 +199,7 @@ function summaryParsing(data: SummaryPayloadType): SummaryParsingType[] {
       toReturn.push({
         label: `Attachment ${i + 1}`,
         value: `<a href='${item.url}'>${item.name}</a>` || 'Unknown',
-        editStepNumber: toReturn.length + 1,
+        editStepNumber: 5,
```

Comment:
I believe that this won't work. The second question defines the number of questions that appear after. This means that the file upload, is NOT always on step 5. You have to check/store the number of steps that exists and then, when pushing the files steps, you always point to that last one step number.